babel.config.js

```diff
@@ -7,7 +7,10 @@ module.exports = (api) => {
   return {
     presets: [
       ['@babel/preset-env', {
-        include: ['@babel/plugin-proposal-nullish-coalescing-operator'],
+        include: [
+          '@babel/plugin-proposal-nullish-coalescing-operator',
+          '@babel/plugin-proposal-optional-chaining',
```

Comment:
```suggestion
          '@babel/plugin-proposal-optional-chaining', // can be removed in a future semver-major. See https://github.com/transloadit/uppy/pull/3335
```

Reply:
Both plugins would be removed, both features are supported in all supported browsers.